org.flathub.sleek.yml

```diff
@@ -0,0 +1,57 @@
+app-id: org.flathub.sleek
```

Comment:
I don't think that's a proper app-id. Something like `com.github.ransome1.sleek` might work.

Reply:
changed it according to your suggestion